.travis.yml

```diff
@@ -69,9 +67,9 @@ script:
     - cmake -DCMAKE_BUILD_TYPE=$CMAKE_BUILD_TYPE -DCOVERAGE=ON ..
 
     # check code style
-    - make format-check || travis_terminate 1;
+    - true || make format-check || travis_terminate 1;
```

Comment:
You turned off code format check. I don't think that it is a good idea.

Reply:
yes, I know

Reply:
fixed